README.md

```diff
@@ -1,9 +1,82 @@
 # dccmonitor
 
-This package is intended to assist data curators to check the status of metadata and documentation files uploaded via the [dccvalidator](https://sage-bionetworks.github.io/dccvalidator/) shiny application.
+This package is intended to assist Sage Bionetworks data curators to check the status of metadata and documentation files uploaded via the [dccvalidator](https://sage-bionetworks.github.io/dccvalidator/) shiny application. dccmonitor is primarily a shiny application, with the ability to use functions for gathering validation information using the package functions only, if desired.
```

Comment:
The intention is to describe the function of the package. I added that bit because it's not required to use the shiny app at all. If someone really loved using R in the terminal and printing validation results to the console, they could do exactly that, using the package functions to gather the validation information. The package has a shiny application, but the package is not only a shiny application, if that makes sense.

Would it be more clear to say, "The dccmonitor package contains functions to gather validation information and a shiny application to view the results", instead?

Reply:
got it! I would say "The dccmonitor package contains functions to gather validation information. Optionally, a shiny application is also available to view the results."